assets/app/views/builds.html

```diff
@@ -192,52 +193,52 @@
               </span>
             </span>
           </div>          
-          <div ng-if="buildConfig.parameters.strategy.type != build.parameters.strategy.type">
-            <div>Build strategy: {{build.parameters.strategy.type}}</div>
+          <div ng-if="buildConfig.spec.strategy.type != build.spec.strategy.type">
+            <div>Build strategy: {{build.spec.strategy.type}}</div>
           </div>
-          <div ng-switch="build.parameters.strategy.type">
-            <div ng-switch-when="STI" ng-if="build.parameters.strategy.stiStrategy.image">
-              Builder image: {{build.parameters.strategy.stiStrategy.image}}
+          <div ng-switch="build.spec.strategy.type">
+            <div ng-switch-when="Source" ng-if="build.spec.strategy.sourceStrategy.from.kind == 'DockerImage'">
+              Builder image: {{build.spec.strategy.sourceStrategy.from.name}}
             </div>
             <div ng-switch-when="Docker">
-              <div ng-if="build.parameters.strategy.dockerStrategy.image">
-                Builder image: {{build.parameters.strategy.dockerStrategy.image}}
+              <div ng-if="build.spec.strategy.dockerStrategy.from.name">
+                Builder image: {{build.spec.strategy.dockerStrategy.from.name}}
               </div>
             </div>
-            <div ng-switch-when="Custom" ng-if="build.parameters.strategy.customStrategy.image">
-              Builder image: {{build.parameters.strategy.customStrategy.image}}
+            <div ng-switch-when="Custom" ng-if="build.spec.strategy.customStrategy.from.name">
+              Builder image: {{build.spec.strategy.customStrategy.from.name}}
             </div>
           </div>          
-          <div ng-if="build.parameters.source">
-            <div ng-if="build.parameters.source.type == 'Git'">
-              <div ng-if="buildConfig.parameters.source.git.uri != build.parameters.source.git.uri">
-                Source repo: <span ng-bind-html='build.parameters.source.git.uri | githubLink | linky'></span>
+          <div ng-if="build.spec.source">
+            <div ng-if="build.spec.source.type == 'Git'">
+              <div ng-if="buildConfig.spec.source.git.uri != build.spec.source.git.uri">
+                Source repo: <span ng-bind-html='build.spec.source.git.uri | githubLink | linky'></span>
               </div>
             </div>
             <!-- TODO add git ref details -->
           </div>
-          <div ng-if="build.parameters.output">
-            <div ng-if="buildConfig.parameters.output.to.kind != build.parameters.output.to.kind && buildConfig.parameters.output.to.name != build.parameters.output.to.name">
-              <div ng-switch="build.parameters.output.to.kind">
+          <div ng-if="build.spec.output">
+            <div ng-if="buildConfig.spec.output.to.kind != build.spec.output.to.kind && buildConfig.spec.output.to.name != build.spec.output.to.name">
+              <div ng-switch="build.spec.output.to.kind">
```

Comment:
shouldn't need to switch on kind any more, just do `Output image: {{build.spec.output.to | imageObjectRef}}`